setup.cfg

```diff
@@ -16,6 +16,7 @@ python_requires = >= 3.7
 install_requires =
     datalad >= 0.17
     datalad-container
+    python-debian
```

Comment:
`python3-debian`?

Reply:
No, this is for pip, there is no `python3-debian`